### ISSUE TITLE
Import Issue

### ISSUE THREAD

**Issue by BradRSnipeIT** [grokability/snipe-it#8659]
**Describe the bug**
I have a standalone version of SNIPEIT (not on Internet). When I try to import your sample assets csv, I get the following error:

"Integrity constraint violation: 1048 Column 'requestable' cannot be null..." shows the query and the requestable entry is null.  Not sure where to go from here.

Thanks,
Brad

**Comment by snipe** [comment]
Please provide answers to the questions you deleted.

#### Confirm you have done the following before posting your bug report:

- [ ] I have enabled debug mode 
- [ ] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

**Describe the bug**
A clear and concise description of what the bug is.

**To Reproduce**
Steps to reproduce the behavior:
1. Go to '...'
2. Click on '....'
3. Scroll down to '....'
4. See error

**Expected behavior**
A clear and concise description of what you expected to happen.

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Server (please complete the following information):**
 - Snipe-IT Version 
 - OS: [e.g. Ubuntu, CentOS]
 - Web Server: [e.g. Apache, IIS]
 - PHP Version 

**Desktop (please complete the following information):**
 - OS: [e.g. iOS]
 - Browser [e.g. chrome, safari]
 - Version [e.g. 22]

**Smartphone (please complete the following information):**
 - Device: [e.g. iPhone6]
 - OS: [e.g. iOS8.1]
 - Browser [e.g. stock browser, safari]
 - Version [e.g. 22]

**Additional context**
- Is this a fresh install or an upgrade? 
- What OS and web server you're running Snipe-IT on
- What method you used to install Snipe-IT (install.sh, manual installation, docker, etc)
- Include what you've done so far in the installation, and if you got any error messages along the way.
- Indicate whether or not you've manually edited any data directly in the database

Add any other context about the problem here.

Please do not post an issue without answering the related questions above. If you have opened a different issue and already answered these questions, answer them again, once for every ticket. It will be next to impossible for us to help you.

![enlkq1k](https://user-images.githubusercontent.com/197404/31936238-da7c20cc-b865-11e7-8706-4cf76a7ec461.gif)

**Comment by BradRSnipeIT** [comment]
Please provide answers to the questions you deleted.

Confirm you have done the following before posting your bug report:
 I have enabled debug mode - No, just logged the import
 I have read checked the Common Issues page - Yes
Describe the bug
Describe the bug
I have a standalone version of SNIPEIT (not on Internet). When I try to import your sample assets csv, I get the following error:

"Integrity constraint violation: 1048 Column 'requestable' cannot be null..." shows the query and the requestable entry is null. Not sure where to go from here.  

Thanks,
Brad

To Reproduce
Steps to reproduce the behavior:

Go to '...'
Click on '....'
Scroll down to '....'
See error
Expected behavior
A clear and concise description of what you expected to happen. 

Screenshots
If applicable, add screenshots to help explain your problem.

Server (please complete the following information):

Snipe-IT Version
OS: CentOS
Web Server: Apache
PHP Version: ?
Desktop (please complete the following information):

OS: CentOS - Command Line
Browser [e.g. chrome, safari]
Version [e.g. 22]
Smartphone (please complete the following information):

Device: [e.g. iPhone6]
OS: [e.g. iOS8.1]
Browser [e.g. stock browser, safari]
Version [e.g. 22]
Additional context

Is this a fresh install or an upgrade? Fresh Install
What OS and web server you're running Snipe-IT on: CENTOS, Apache
What method you used to install Snipe-IT (install.sh, manual installation, docker, etc): Manual
Include what you've done so far in the installation, and if you got any error messages along the way.
Indicate whether or not you've manually edited any data directly in the database
Add any other context about the problem here.

**Comment by snipe** [comment]
Do you have a value for "requestable" in your CSV?

**Comment by BradRSnipeIT** [comment]
No, as a matter of fact, when I saw the error, figured that I'd just try to import the example (https://snipeitapp.com/assets/downloads/sample-assets.csv) and it is giving me the same error.

**Comment by snipe** [comment]
What happens when you create a column called Requestable and give it a value of 0 or 1?
